spec/renderer/integration/store/TimelineSpace/Contents/Mentions.spec.js

```diff
@@ -0,0 +1,152 @@
+import Mastodon from 'megalodon'
+import { createLocalVue } from '@vue/test-utils'
+import Vuex from 'vuex'
+import Mentions from '~/src/renderer/store/TimelineSpace/Contents/Mentions'
+
+jest.genMockFromModule('megalodon')
+jest.mock('megalodon')
+
+let state = () => {
+  return {
+    lazyLoading: false,
+    heading: true,
+    mentions: [],
+    unreadMentions: [],
+    filter: ''
+  }
+}
+
+const initStore =() => {
```

Comment:
__[ESLint]__ Error - Infix operators must be spaced. (space-infix-ops) ([link](https://sider.review/gh/repos/124234495/pulls/849#issue-3855130))

<sub>posted by Sider</sub>